mne/tests/test_source_estimate.py

```diff
@@ -381,7 +387,8 @@ def test_io_stc(tmpdir):
 @requires_h5py
 def test_io_stc_h5(tmpdir):
     """Test IO for STC files using HDF5."""
-    for stc in [_fake_stc(), _fake_vec_stc()]:
+    for stc in [_fake_stc(is_complex=True), _fake_vec_stc(is_complex=True),
+                _fake_stc(is_complex=False), _fake_vec_stc(is_complex=False)]:
```

Comment:
you could use pytest parametrize

Reply:
not sure what you mean

Reply:
```
@pytest.mark.parametrize('is_complex', (True, False))
@pytest.mark.parametrize('vector', (True, False))
def test_io_stc_h5(tmpdir, is_complex, vector):
    """Test IO ..."""
    if is_vector:
        stc = _fake_vec_stc(is_complex=is_complex)
    else:
        stc = _fake_stc(is_complex=is_complex)
```
now no need for the `for stc in` loop because `pytest` will do it via the parameterization combinations

Reply:
Thanks - parametrization added! (learning a lot (appreciated))